rq/worker_registration.py

```diff
@@ -9,13 +10,27 @@
 
 from rq.utils import split_list
 
-from .utils import as_text
+from .utils import as_text, utcformat, utcparse
 
 WORKERS_BY_QUEUE_KEY = 'rq:workers:%s'
 REDIS_WORKER_KEYS = 'rq:workers'
+REDIS_WORKER_LAST_CLEANED_AT = 'rq:workers:last_cleaned_at'
 MAX_KEYS = 1000
 
 
+def get_last_cleaned_at(connection: 'Redis') -> Optional[datetime]:
+    """Returns the last time maintenance task was run."""
+    last_cleaned_at = connection.get(REDIS_WORKER_LAST_CLEANED_AT)
+    if last_cleaned_at:
+        return utcparse(as_text(last_cleaned_at))
+    return None
+
+
+def set_last_cleaned_at(last_cleaned_at: datetime, connection: 'Redis'):
+    """Sets the last time maintenance task was run."""
+    connection.set('last_cleaned_at', utcformat(last_cleaned_at))
```

Comment:
Mind storing this as UTC timestamp instead of string?

Reply:
How do I do that?

Reply:
Use [current_timestamp()](https://github.com/rq/rq/blob/e2b600d9dfb3ed834cc766b368a0cb5d56ecfc9c/rq/utils.py#L221)